### ISSUE TITLE
Error on HTML generation from multiple files

### ISSUE THREAD

**Issue by kvark** [christianvoigt/argdown#180]
```
[Error: EEXIST: file already exists, copyfile '/usr/local/lib/node_modules/@argdown/cli/node_modules/@argdown/core/dist/plugins/argdown.css' -> '/Users/dmalyshau/Code/webgpu-debate/html/argdown.css'] {
  errno: -17,
  code: 'EEXIST',
  syscall: 'copyfile',
  path: '/usr/local/lib/node_modules/@argdown/cli/node_modules/@argdown/core/dist/plugins/argdown.css',
  dest: '/Users/dmalyshau/Code/webgpu-debate/html/argdown.css'
}
```
Looks like it's trying to copy the CSS for each argdown file, and then bites its own tail?

**Comment by kvark** [comment]
See full log in https://github.com/kvark/webgpu-debate/runs/954040535?check_suite_focus=true

**Comment by christianvoigt** [comment]
Thanks again, we missed this issue completely because we never really used the html export but used highlighted source code instead (like the source view in the web component) as it had the advantage to teach our students the Argdown syntax. Is fixed in v1.5.2, I also changed the html export test to export multiple files.

**Comment by kvark** [comment]
Thank you for fixing this promptly!
